proxy/director.go

```diff
@@ -29,9 +29,29 @@ func newDirector(urlsFunc GetProxyURLs, failureWait time.Duration, refreshInterv
 	}
 	d.refresh()
 	go func() {
+		// In order to prevent missing proxy endpoints in the first try:
+		// when given refresh interval of 30-second or greater
+		// and where there is no available proxy endpoints yet,
+		// give 1-second refreshInterval.
 		for {
+			ri := refreshInterval
+			if ri >= 30*time.Second {
```

Comment:
This number is a bit magic. Could you make it a const and reference it's derived from the default config (i.e. proxyRefreshIntervalMs) perhaps?